2022-06-02 19:20:47,860 INFO    MainThread:9521 [wandb_setup.py:_flush():76] Loading settings from /home/diego/.config/wandb/settings
2022-06-02 19:20:47,860 INFO    MainThread:9521 [wandb_setup.py:_flush():76] Loading settings from /home/diego/machine learning/machineLearnig_project1/Machine-learning-project-of-heart-with-MLops/source/notebooks/wandb/settings
2022-06-02 19:20:47,860 WARNING MainThread:9521 [wandb_setup.py:_flush():76] Unknown environment variable: WANDB_API
2022-06-02 19:20:47,860 INFO    MainThread:9521 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-06-02 19:20:47,860 INFO    MainThread:9521 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-06-02 19:20:47,860 INFO    MainThread:9521 [wandb_init.py:_log_setup():428] Logging user logs to /home/diego/machine learning/machineLearnig_project1/Machine-learning-project-of-heart-with-MLops/source/notebooks/wandb/run-20220602_192047-n6nmhkrq/logs/debug.log
2022-06-02 19:20:47,860 INFO    MainThread:9521 [wandb_init.py:_log_setup():429] Logging internal logs to /home/diego/machine learning/machineLearnig_project1/Machine-learning-project-of-heart-with-MLops/source/notebooks/wandb/run-20220602_192047-n6nmhkrq/logs/debug-internal.log
2022-06-02 19:20:47,860 INFO    MainThread:9521 [wandb_init.py:_jupyter_setup():378] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f471e9e7580>
2022-06-02 19:20:47,860 INFO    MainThread:9521 [wandb_init.py:init():462] calling init triggers
2022-06-02 19:20:47,860 INFO    MainThread:9521 [wandb_init.py:init():465] wandb.init called with sweep_config: {}
config: {}
2022-06-02 19:20:47,861 INFO    MainThread:9521 [wandb_init.py:init():515] starting backend
2022-06-02 19:20:47,861 INFO    MainThread:9521 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-06-02 19:20:47,863 INFO    MainThread:9521 [backend.py:ensure_launched():217] starting backend process...
2022-06-02 19:20:47,866 INFO    MainThread:9521 [backend.py:ensure_launched():222] started backend process with pid: 9621
2022-06-02 19:20:47,867 INFO    MainThread:9521 [wandb_init.py:init():525] backend started and connected
2022-06-02 19:20:47,912 INFO    MainThread:9521 [wandb_run.py:_label_probe_notebook():1089] probe notebook
2022-06-02 19:20:47,913 INFO    MainThread:9521 [wandb_run.py:_label_probe_notebook():1099] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-06-02 19:20:47,913 INFO    MainThread:9521 [wandb_init.py:init():596] updated telemetry
2022-06-02 19:20:47,921 INFO    MainThread:9521 [wandb_init.py:init():627] communicating run to backend with 30 second timeout
2022-06-02 19:20:49,030 INFO    MainThread:9521 [wandb_run.py:_on_init():1924] communicating current version
2022-06-02 19:20:49,352 INFO    MainThread:9521 [wandb_run.py:_on_init():1928] got version response 
2022-06-02 19:20:49,352 INFO    MainThread:9521 [wandb_init.py:init():662] starting run threads in backend
2022-06-02 19:20:50,325 INFO    MainThread:9521 [wandb_run.py:_console_start():1898] atexit reg
2022-06-02 19:20:50,327 INFO    MainThread:9521 [wandb_run.py:_redirect():1771] redirect: SettingsConsole.WRAP
2022-06-02 19:20:50,327 INFO    MainThread:9521 [wandb_run.py:_redirect():1808] Wrapping output streams.
2022-06-02 19:20:50,328 INFO    MainThread:9521 [wandb_run.py:_redirect():1832] Redirects installed.
2022-06-02 19:20:50,328 INFO    MainThread:9521 [wandb_init.py:init():688] run started, returning control to user process
2022-06-02 19:20:52,437 INFO    MainThread:9521 [wandb_init.py:_pause_backend():341] pausing backend
2022-06-02 19:20:52,437 INFO    MainThread:9521 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-06-02 19:20:54,846 INFO    MainThread:9521 [wandb_init.py:_resume_backend():350] resuming backend
2022-06-02 19:20:54,850 INFO    MainThread:9521 [wandb_init.py:_pause_backend():341] pausing backend
2022-06-02 19:20:54,850 INFO    MainThread:9521 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-06-02 19:23:51,509 INFO    MainThread:9521 [wandb_init.py:_resume_backend():350] resuming backend
2022-06-02 19:23:51,532 INFO    MainThread:9521 [wandb_setup.py:_flush():76] Loading settings from /home/diego/.config/wandb/settings
2022-06-02 19:23:51,532 INFO    MainThread:9521 [wandb_setup.py:_flush():76] Loading settings from /home/diego/machine learning/machineLearnig_project1/Machine-learning-project-of-heart-with-MLops/source/notebooks/wandb/settings
2022-06-02 19:23:51,532 WARNING MainThread:9521 [wandb_setup.py:_flush():76] Unknown environment variable: WANDB_API
2022-06-02 19:23:51,532 INFO    MainThread:9521 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-06-02 19:23:51,532 INFO    MainThread:9521 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-06-02 19:23:51,533 INFO    MainThread:9521 [wandb_init.py:_log_setup():428] Logging user logs to /home/diego/machine learning/machineLearnig_project1/Machine-learning-project-of-heart-with-MLops/source/notebooks/wandb/run-20220602_192351-glwojvop/logs/debug.log
2022-06-02 19:23:51,533 INFO    MainThread:9521 [wandb_init.py:_log_setup():429] Logging internal logs to /home/diego/machine learning/machineLearnig_project1/Machine-learning-project-of-heart-with-MLops/source/notebooks/wandb/run-20220602_192351-glwojvop/logs/debug-internal.log
2022-06-02 19:23:51,533 INFO    MainThread:9521 [wandb_init.py:init():462] calling init triggers
2022-06-02 19:23:51,534 INFO    MainThread:9521 [wandb_init.py:init():465] wandb.init called with sweep_config: {}
config: {}
2022-06-02 19:23:51,534 INFO    MainThread:9521 [wandb_init.py:init():481] re-initializing run, found existing run on stack: n6nmhkrq
2022-06-02 19:23:51,536 INFO    MainThread:9521 [wandb_run.py:_finish():1685] finishing run diego25rm/project_heart/n6nmhkrq
2022-06-02 19:23:51,536 INFO    MainThread:9521 [jupyter.py:save_history():435] not saving jupyter history
2022-06-02 19:23:51,537 INFO    MainThread:9521 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-06-02 19:23:51,537 INFO    MainThread:9521 [wandb_init.py:_jupyter_teardown():360] cleaning up jupyter logic
2022-06-02 19:23:51,537 INFO    MainThread:9521 [wandb_run.py:_atexit_cleanup():1867] got exitcode: 0
2022-06-02 19:23:51,538 INFO    MainThread:9521 [wandb_run.py:_restore():1839] restore
2022-06-02 19:23:52,091 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 928
  total_bytes: 928
}

2022-06-02 19:23:52,767 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 928
  total_bytes: 928
}

2022-06-02 19:23:53,131 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 928
  total_bytes: 4104
}

2022-06-02 19:23:53,234 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 928
  total_bytes: 4104
}

2022-06-02 19:23:53,337 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 928
  total_bytes: 4104
}

2022-06-02 19:23:53,439 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 928
  total_bytes: 4104
}

2022-06-02 19:23:53,544 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 928
  total_bytes: 4104
}

2022-06-02 19:23:53,646 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 928
  total_bytes: 4104
}

2022-06-02 19:23:53,748 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 928
  total_bytes: 4104
}

2022-06-02 19:23:53,850 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4104
  total_bytes: 4104
}

2022-06-02 19:23:53,954 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4104
  total_bytes: 4104
}

2022-06-02 19:23:54,057 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4104
  total_bytes: 4104
}

2022-06-02 19:23:54,159 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4104
  total_bytes: 4104
}

2022-06-02 19:23:54,262 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4104
  total_bytes: 4104
}

2022-06-02 19:23:54,364 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4104
  total_bytes: 4104
}

2022-06-02 19:23:55,126 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4104
  total_bytes: 4104
}

2022-06-02 19:23:56,231 INFO    MainThread:9521 [wandb_run.py:_on_finish():1996] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4104
  total_bytes: 4104
}
local_info {
}

2022-06-02 19:23:57,950 INFO    MainThread:9521 [wandb_run.py:_footer_history_summary_info():3104] rendering history
2022-06-02 19:23:57,951 INFO    MainThread:9521 [wandb_run.py:_footer_history_summary_info():3136] rendering summary
2022-06-02 19:23:57,955 INFO    MainThread:9521 [wandb_run.py:_footer_sync_info():3060] logging synced files
